common/src/main/java/org/keycloak/common/util/KeystoreUtil.java

```diff
@@ -38,8 +39,8 @@
 public class KeystoreUtil {
 
     public enum KeystoreFormat {
-        JKS("jks"),
-        PKCS12("p12", "pfx"),
+        JKS("jks", "truststore"),
```

Comment:
We should be more explicit and include here also the `.keystore`, right?
```suggestion
        JKS("jks", "truststore", "keystore"),
```

Reply:
> We should be more explicit and include here also the .keystore, right?

I didn't include it because there's some potential confusion there. We already use .keystore for the default server keystore. That is in pkcs12 format, not jks. However it still works as pkcs12 even when quarkus detects the type as jks because the jdk will try both formats when you specify jks. Since we don't rely on the KeystoreUtil to define keystores, I just left that out for now.

Reply:
Actually we could have the same concern with .truststore - because that's what we're using for the generated keycloak.truststore.

Reply:
> Actually we could have the same concern with .truststore

@shawkins yes, that's exactly what I wanted to point out as well xD 

Like, even when the `KeystoreFormat` is barely used right now, we could move it out of the util class and have its own lifecycle with a proper setting. 

Just right now, we state in docs that `.keystore` and `.truststore` extensions are considered JKS, but it's not reflected here. 

As it is an inner enum of the util class, I would not consider it as a big deal to have it "complete", as the source code should not depend on it in a big manner (IMHO).

So, I'll leave it up to you what to include there.

Reply:
I primarly wanted to align the pkcs12 extensions, so I'll just leave out the .truststore and .keystore conventions for now.

Reply:
Agree